### PR TITLE
Least Common Multiple

### PR DESCRIPTION
**Pull request by yashaswibiyahut**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Least Common Multiple using gcd

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x]  I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#1300

### CODE REVIEW

**Comment by yashaswibiyahut** [comment]
kindly review this PR @Kajol-Kumari @SKAUL05 @siddharth25pandey @iamrajiv @HarshCasper

**Review by SKAUL05 — APPROVED** [review]
LGTM

**Comment by yashaswibiyahut** [comment]
@SKAUL05 if one 1 review is enough then  kindly merge this PR 😊

File: C-Plus-Plus/other/lcm.cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
+//Using GCD, find LCM of two number.
+
```
Comment (atarax665) [inline C-Plus-Plus/other/lcm.cpp:2]: You may add description of the in-built LCM function that was introduced in C++17, which may come handy during competitive programming. You can read about it [here](https://www.geeksforgeeks.org/stdlcm-in-cpp17/)

File: C-Plus-Plus/other/lcm.cpp [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+{
+	//return Lcm of two number
+    return (a / gcd(a, b)) * b;
+}
+
+//main function
+int main()
+{
+	//Input 2 numbers for Lcm
+    int n1, n2, max;
```
Comment (atarax665) [inline C-Plus-Plus/other/lcm.cpp]:
```suggestion
    int n1, n2;
```
We don't need any extra variable here.

File: C-Plus-Plus/other/lcm.cpp [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    cout << "Enter two numbers: ";
+    cin >> n1 >> n2;
+    //Print output
+    cout<<"\tOUTPUT\n";
```
Comment (atarax665) [inline C-Plus-Plus/other/lcm.cpp:30]: Remove this line, its better to print the output as ```LCM of a and b --> x```

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by yashaswibiyahut** [comment]
done @atarax665  kindly review it.

**Review by vatsalkesarwani12 — APPROVED** [review]
👍🏻

**Review by iamrajiv — APPROVED** [review]

**Comment by yashaswibiyahut** [comment]
Kindly merge this PR @iamrajiv

**Review by atarax665 — APPROVED** [review]

**Pull request merged**